site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -230,6 +262,21 @@ The time in seconds until a setup password Id is no longer valid and cannot be u
 [field]#tenant.externalIdentifierConfiguration.twoFactorIdTimeToLiveInSeconds# [type]#[Integer]# [required]#Required# [since]#Available since 1.8.0#::
 The time in seconds until a two factor Id is no longer valid and cannot be used by the Two Factor Login API. Value must be greater than 0.
 
+[field]#tenant.externalIdentifierConfiguration.twoFactorOneTimeCodeIdGenerator.length# [type]#[Integer]# [optional]#Optional# [default]#6# [since]#Available since 1.8.0#::
++
+:generator_type: twoFactorOneTimeCodeIdGenerator
+:generator_display_name: the two factor code Id
+include::_tenant-generator-length-config-rules.adoc[]
+:generator_type!:
+:generator_display_name!:
++
+
+[field]#tenant.externalIdentifierConfiguration.twoFactorOneTimeCodeIdGenerator.type# [type]#[String]# [optional]#Optional# [default]#randomDigits# [since]#Available since 1.8.0#::
```

Comment:
Was this really available since 1.8? or is that copy pasta?

Reply:
Updated to 1.27 which I believe is accurate